src/Components/ResultPage/ResultPage.js

```diff
@@ -14,89 +61,70 @@ export const ResultPage = () => {
+              || props.randomActivity.name
+              }/details`
+              }>
+             <button className="button pick-activity">Choose me!</button>
+          </Link>}
+            <button 
+              onClick={props.determineRandomActivity}
+              className="button skip-activity">{props.error ? "back" : "skip"}</button>
           </div>
 
           <div className="the-rest-container">
-            <div className="inner-all-rest">
-              <div className="border">
-                <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
-            <div className="inner-all-rest">
-              <div className="border">
-                <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
-            <div className="inner-all-rest">
-              <div className="border">
-                <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
-            <div className="inner-all-rest">
-              <div className="border">
-              <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
-            <div className="inner-all-rest">
-              <div className="border">
-              <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
-            <div className="inner-all-rest">
-              <div className="border">
-                <GoLightBulb className="bulb-icon"/>
-                <h3 className="title">activity name</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-                <h3 className="title">more information</h3>
-              </div>
-            </div>
+           {displayActivityGallery()}
```

Comment:
We should change this from a method to a variable, for `displayActivityGallery`.